Http client engine not provived

Hello. 
We are using HTTPS uri schema in pid issuer.
It's impossible to validate this document, there is no http client engine on classpath.

Caused by: java.lang.IllegalStateException: Failed to find HTTP client engine implementation in the classpath: consider adding client engine dependency. See https://ktor.io/docs/http-client-engines.html	
		at eu.europa.ec.eudi.sdjwt.vc.SdJwtVcVerifierKt.issuerJwkSource(SdJwtVcVerifier.kt:323) ~[eudi-lib-jvm-sdjwt-kt-0.10.0.jar:0.10.0]

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -51,6 +51,9 @@ dependencies {
         because("required by walt.id")
     }
     implementation(libs.sd.jwt)
+    implementation(libs.ktor.client.java) {
+        because("ktor client engine to use (required by SdJwtVcVerifier)")
+    }
 
     testImplementation(kotlin("test"))
     testImplementation(libs.kotlinx.coroutines.test)

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -18,6 +18,7 @@ jacoco = "0.8.11"
 swaggerUi = "5.18.2"
 waltid = "0.9.0"
 sdJwt = "0.10.0"
+ktor = "3.0.1"
 
 [libraries]
 kotlinx-coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "coroutines" }
@@ -30,6 +31,7 @@ arrow-fx-coroutines = { module = "io.arrow-kt:arrow-fx-coroutines", version.ref 
 swagger-ui = { module = "org.webjars:swagger-ui", version.ref = "swaggerUi" }
 waltid-mdoc-credentials = { module = "id.walt.mdoc-credentials:waltid-mdoc-credentials-jvm", version.ref = "waltid" }
 sd-jwt = { module = "eu.europa.ec.eudi:eudi-lib-jvm-sdjwt-kt", version.ref = "sdJwt" }
+ktor-client-java = { module = "io.ktor:ktor-client-java", version.ref = "ktor" }
 
 [plugins]
 foojay-resolver-convention = { id = "org.gradle.toolchains.foojay-resolver-convention", version.ref = "foojay" }

